std_msgs/Header header

sensor_msgs/Image mask
sensor_msgs/Image masked_depth

string class_name
float32 class_dist
float32 class_conf

